components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -182,6 +183,7 @@ public class ApiMgtDAO {
     private boolean forceCaseInsensitiveComparisons = false;
     private boolean multiGroupAppSharingEnabled = false;
     private String KeyManagerAccessPublic = "PUBLIC";
+    private String GatewayAccessPublic = "PUBLIC";
```

Comment:
Can't we use the Constant defined before for this, Since this is kind of a duplication of a variable

Reply:
Fixed.